src/utils/_colors.scss

```diff
@@ -34,3 +38,15 @@
     @return mix(white, $color, nth($step-variations-positive, $step));
```

Comment:
As you said David, we cannot use this mixing either

Reply:
Please explain. I don't get it.

Reply:
The ColorVariations function defined in helper folder is already using this `@mix` mixin so it makes no sense to use it here as standalone. In addition we cannot use our own random values to make a color darker or lighter. That's why we have the `color-variations` func